### PR TITLE
Preview page for a service in the backoffice

### LINKED ISSUE #1066: Service preview in the backoffice
**Is your feature request related to a problem? Please describe.**
There is need to make a service preview in the backoffice

**Describe the solution you'd like**
We can add a button for preview redirecting to the page which is a preview of the service in the backoffice.
In the preview page should have "back to edit" and "save service" buttons

### PR DESCRIPTION
**Pull request by goreck888**
Add preview of service in the marketplace view.

Fixes #1066

### CODE REVIEW

**Comment by mkasztelnik** [comment]
The preliminary version is created based on @goreck888 changes. @agpul, @roksanaer, @MinLidia you can play with it and post your comments (most probably these comments will be connected with how ugly top buttons look like - I see you @abacz :smile:). 

There are some elements which are tricky:
  * `Ask a question about this service` -> right now this section is not shown in the preview
  * Order button - it is like in the final view - it starts the order process. Most probably it should behave differently on the preview
  * All service links give a possibility to be clicked. This is tricky since a user can click a link and loose introduced changes (right now it is not possible to go back using browser back button). Maybe this is good enough for now?
  * Rating link is shown in the header only when service is persisted
  * Rating tab is disabled and cannot be clicked

For now, the following elements are not working, need to be added/improved:
  * [ ] Logo can not be added to the preview. Most probably I need to write this file into a temporary directory and fetch it from there while saving preview
  * [ ] Tests are missing
  * [ ] Code cleanup

**Comment by agpul** [comment]
I've started checking this functionality and I request for first change:
1. I create the service in Backoffice
2. I fill some fields
3. I click Preview button
4. I click then Go back to edit
5. All of fields which were filled before are empty now.

**Comment by agpul** [comment]
Coming back from preview--> new is OK now but I have other problems.
First one:
1. Create a new service
2. Fill all mandatory fields
3. Click Create button and see:
image

Second one:
1. Create new service and fill only title
2. Click Preview
3. In a preview view click Confirm changes
4.  Review below problems (this behavior is OK):
image
5. Fill lacking fileds (mark red)
6. Click Preview
7. Click Confirm changes one more time
8. Even if all mandatory fields are filled the form ask me to  "Review below problmes"

**Comment by mkasztelnik** [comment]
All mentioned problems are solved, this is ready to be reviewed.

**Comment by kmarszalek** [comment]
flat logo getting stretched
Alien and Invasive Species Virtual Research Environment
Zrzut ekranu 2019-09-30 o 12 07 30

File: app/controllers/backoffice/services/logo_previews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      authorize(@service)
+    end
+
+    show_logo_preview
+  end
+
+  private
+
+    def show_logo_preview
+      puts "service id: #{params[:service_id]}"
```
Comment (kmarszalek) [inline app/controllers/backoffice/services/logo_previews_controller.rb]: what for puts?

**Pull request merged**